Dockerfile

```diff
@@ -41,7 +41,9 @@ RUN dnf upgrade -y \
     chromedriver-$CHROMIUM_VERSION \
     chromium-$CHROMIUM_VERSION \
     firefox-$FIREFOX_VERSION \
-    python2-pip-$PYTHON_PIP_VERSION \
+    python3-pip-$PYTHON_PIP_VERSION \
+	python3-crypto-2.6.1* \
+	python3-devel-3.7.3* \
```

Comment:
Why are the crypto and devel dependencies required?

Reply:
crypto might not be, i forget why that is there. devel is the headers for the c that libraries in python may need

Reply:
I don't really like the "may need". Python didn't need devel in version 2, why would it need it in version 3?

Reply:
i seem to remember it being needed, i will remove and see